src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/InnerApi/CandidateApi/Requests/PostApplicationsCountApiRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.FindAnApprenticeship.Domain.Models;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+
+namespace SFA.DAS.FindAnApprenticeship.InnerApi.CandidateApi.Requests
+{
+    public record GetApplicationsCountApiRequest(Guid CandidateId, ApplicationStatus Status) : IGetApiRequest
```

Comment:
Can we rename the filename to match please?

Reply:
Renamed the file as appropiate